velox/exec/HashTable.h

```diff
@@ -74,7 +77,8 @@ struct HashLookup {
   /// Input to groupProbe and joinProbe APIs.
 
   /// Set of row numbers of row to probe.
-  raw_vector<vector_size_t> rows;
+  raw_vector<vector_size_t, AlignedStlAllocator<vector_size_t, simd::kPadding>>
```

Comment:
Maybe give this thing an alias since you are going to create a lot of these.  Something like `HsaRawVector`